src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -416,6 +416,81 @@ public void testSetFormatsByArgumentIndex() {
         emf.setFormatsByArgumentIndex(new Format[]{new LowerCaseFormat(), new UpperCaseFormat()});
     }
 
+    @Test
+    public void testFailsToCreateExtendedMessageFormatTakingThreeArgumentsThrowsIllegalArgumentExceptionOne() {
+        Map<String, FormatFactory> map = new HashMap<String, FormatFactory>();
+        ExtendedMessageFormat extendedMessageFormat = null;
```

Comment:
No need to create a variable if it is not used. This will produce unused variable warnings in IDEs etc.